cou/utils/juju_utils.py

```diff
@@ -442,6 +442,21 @@ async def get_status(self) -> FullStatus:
         model = await self._get_model()
         return await model.get_status()
 
+    async def update_status(self, unit: str) -> None:
+        """Run the update_status hook on the given unit if the hook exists.
+
+        :param unit: Name of the unit to run update-status hook
+        :type unit: str
+        :raises CommandRunFailed: When update-status hook failed
+        """
+        update_status_hook = "hooks/update-status"
+        try:
+            await self.run_on_unit(unit, f"ls {update_status_hook}")
+        except CommandRunFailed:
+            logger.warning("Skipped running '%s': file does not exist", update_status_hook)
```

Comment:
**suggestion (non-blocking):** 

The message missing file can not give enough information compare to `Skipped running because update-status hook is missing`. Consider the situation that user don't know the details how update-status hook is triggered.

Reply:
make sense, maybe `logger.debug` ?

Reply:
Sure.